### ISSUE TITLE
Bug installing media bundle

### ISSUE THREAD

**Issue by mysterty** [sonata-project/SonataMediaBundle#1084]
Hello, i'm trying to install sonata media bundle (with classification), but when i try to run doctrine:schema:update, i have this error :

``` bash
./bin/console doctrine:schema:update


  [Doctrine\ORM\Mapping\MappingException]                                                                                                               
  The target-entity Application\Sonata\ClassificationBundle\Entity\Category cannot be found in 'Application\Sonata\MediaBundle\Entity\Media#category'.
```

**Comment by greg0ire** [comment]
This class is supposed to be generated thanks to easy extends bundle. What command from easy extends did you run?

**Comment by Soullivaneuh** [comment]
Please fill GitHub template for bug report.

**Comment by mysterty** [comment]
@greg0ire 
php binconsole sonata:easy-extends:generate --dest=src SonataClassificationBundle

But there is still many things to handle after, and i have no more time to go farther.
I just can warn you there is problem somewhere :/
@Soullivaneuh : sorry, i'm running out of time today on my own work :/

**Comment by greg0ire** [comment]
Ok, when you have time, tell us what files where generated by this command (is there a Category class or not ?)

**Comment by Soullivaneuh** [comment]
>  sorry, i'm running out of time today on my own work :/

We understand but please understand that we don't have lot of time either to ask people to do it.

Next time, just wait having time for that before opening an issue. Thanks.

**Comment by greg0ire** [comment]
Related : http://stackoverflow.com/questions/38568753/sonata-media-bundle-missing-entity-category

**Comment by mysterty** [comment]
Hello,

I tried again. Problem was with sonata_classification.yml wich i didn't create at first time, but i still had issue in the admin :
`You have requested a non-existent service "request".`
I had to abandon your MediaBundle to make my own solution, but i hope this report will help your project to become better ;)

**Comment by greg0ire** [comment]
Looks like there is indeed a bug right here : https://github.com/sonata-project/SonataMediaBundle/blob/3.x/Controller/MediaAdminController.php#L32

**Comment by greg0ire** [comment]
Related : #1068

**Comment by jordisala1991** [comment]
Fixed on latest release of MediaBundle 3.4.0
